### DISCUSSION TITLE
realtime data inside extend with JS

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#3387] [Q&A] [answered]
is there any way to use realtime functionality inside pb_hooks folder (extend with JS)

**Comment by ganigeorgiev** [comment, marked as answer]
It depends what functionality you are looking for. Could you elaborate what is your use case and what you are trying to do?

If you just want to listen to model changes and execute some server-side action, you can attach to the specific event hook - https://pocketbase.io/docs/js-event-hooks/ (_most of the realtime API events are triggered by the `OnModel*` hooks_).
